packages/terra-clinical-onset-picker/src/OnsetPicker.jsx

```diff
@@ -174,20 +174,27 @@ class OnsetPicker extends React.Component {
    * @param {event} - Triggered change event
    */
   changeAge(event) {
-    const age = Number(event.target.value);
-
-    this.setState((prevState) => {
+    if (event.target.value !== '') {
```

Comment:
can we check this using 'Undefined' or null ? is this value be always be ' ' ? if yes, then declare it as const such as

const BlankValue = ' '; in the beginning of the file. so that it can be reused across the class.

Reply:
The empty string is false-y anyway, so you should be able to cover all cases with `if (event.target.value) {}`

Reply:
Resolved in 7afc761